README.md

```diff
@@ -78,8 +78,6 @@ and build at your side. After building, please replace them.
 [fftw](http://www.fftw.org/).
 [flatcc](https://github.com/dvidelabs/flatcc).
 
-Remark: please add compile options:-fPIC  
-
```

Comment:
Hi Jzern~
Thanks for your reviewing,
Actually, this comment "Remark: please add compile options:-fPIC" is added for building third part library(fftw flatcc),
not for libiac.
Static fftw flatcc libraries are required to be linked into libiac.

Reply:
I see. I might have scanned the README too quickly.

Reply:
I reverted this and did a force push to keep it simple since there's some trailing whitespace I didn't want to add back in with an explicit commit. It's possible the indent could be improved to associate the comment a little better with the list item it's referring to.

Reply:
I agree with you. I will improve the indent.